terraform/internal/rancher/fleet/bundle/files/bundle.yaml

```diff
@@ -11,6 +11,6 @@ spec:
   - clusterName: ${cluster_name}
   dependsOn: ${jsonencode(depends_on)}
   resources:
-  - name: ${name}.yaml
+  - name: templates/${name}.yaml
```

Comment:
@HarrisonWAffel this was the change that made webhooks work

Reply:
ref: https://github.com/rancher/fleet/blob/af889828c835808184322a401fcac506696c6da0/internal/helmdeployer/render/helm.go#L73C1-L108